cmd/ls.go

```diff
@@ -36,10 +36,7 @@ func formatListEntries(apiPath string, ls []apitypes.ListEntry) string {
 	table := make([][]string, len(ls))
 	for i, entry := range ls {
 		var ctimeStr string
-		if err, ok := entry.Errors["attributes"]; ok {
-			cmdutil.ErrPrintf("errored on %v: could not retrieve the attributes: %v\n", entry.Name, err)
-			ctimeStr = "<unknown>"
-		} else if entry.Attributes.Ctime.IsZero() {
+		if entry.Attributes.Ctime == nil {
```

Comment:
`wash ls` is pretty slow compared with `ls` (~4s to list S3 buckets using Michael's account). I think the latency's because of FUSE having to do a find to get the xattrs. If so, it's probably good to remove the xattr dependency soon.

Reply:
It's a theory worth testing. I wouldn't have expected it to be slower because the query to get xattrs should result in later activity being cached.